packages/components/ng-log-to-customer/src/template.html

```diff
@@ -2,6 +2,18 @@
     <span data-ng-bind="$ctrl.errorMessage"></span>
 </oui-message>
 
+<div class="my-2" v-if="$ctrl.kinds.length > 1">
```

Comment:
```suggestion
<div class="my-2" data-ng-if="$ctrl.kinds.length > 1">
```
v-if it's for vueJS i think